iaso/api/org_unit_change_requests/filters.py

```diff
@@ -16,11 +17,11 @@ class Meta:
 
 
 class OrgUnitChangeRequestListFilter(django_filters.rest_framework.FilterSet):
-    org_unit_id = django_filters.NumberFilter(field_name="org_unit_id")
-    org_unit_type_id = django_filters.NumberFilter(method="filter_org_unit_type_id")
-    parent_id = django_filters.NumberFilter(method="filter_parent_id")
-    groups = django_filters.CharFilter(method="filter_groups")
-    project = django_filters.NumberFilter(field_name="org_unit__org_unit_type__projects")
+    org_unit_id = django_filters.NumberFilter(field_name="org_unit_id", label=_("Org unit ID"))
+    org_unit_type_id = django_filters.NumberFilter(method="filter_org_unit_type_id", label=_("Org unit type ID"))
+    parent_id = django_filters.NumberFilter(method="filter_parent_id", label=_("Parent ID"))
+    groups = django_filters.CharFilter(method="filter_groups", label=_("Groups IDs (comma-separated)"))
+    project = django_filters.NumberFilter(field_name="org_unit__org_unit_type__projects", label=_("Project ID"))
```

Comment:
This fixes the UI:

![filters](https://github.com/BLSQ/iaso/assets/281139/896ac64c-f79b-4a3f-a445-5b3892105d1c)